tests/functional/test_metrics.py

```diff
@@ -30,23 +32,23 @@ def reset_metric_set():
 
 
 @pytest.fixture
-def metric() -> Dict[str, str]:
-    return {"name": "single_metric", "unit": MetricUnit.Count, "value": 1}
+def metric() -> Dict[str, Union[str, int]]:
+    return {"name": "single_metric", "unit": MetricUnit.Count, "value": 1, "resolution": MetricResolution.Standard}
```

Comment:
let's revert the `resolution` part and create a new fixture with resolution, so we can correlate a change to a possible breaking change

Reply:
Thanks for the feedback, Hector! Yes, it makes a lot of sense and I reverse it.

I've created new tests to make sure we're not breaking anything.